### PR TITLE
[SD-2904][SD-2905] - Add non-digital stories to the packages

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/tests/authoring_spec.js [hunk 1]
```diff
@@ -877,4 +877,50 @@ describe('authoring actions', function() {
...
+                'publish': true,
+                'correct': true,
+                'kill': true,
+                'package_item': false
+            };
+
+            privileges.setUserPrivileges(userPrivileges);
+            $rootScope.$digest();
+            var itemActions = authoring.itemActions(item);
+            console.log('ACTIONS:', itemActions);
```
Comment (mdhaman) [inline client/app/scripts/superdesk-authoring/tests/authoring_spec.js]: remove this console.log

**Comment by mdhaman** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Comment by plamut** [comment]
Looks good. :+1:

**Pull request merged**